x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/exporter/http/TemplateHttpResource.java

```diff
@@ -88,8 +88,11 @@ protected void doCheck(final RestClient client, final ActionListener<Boolean> li
      */
     @Override
     protected void doPublish(final RestClient client, final ActionListener<Boolean> listener) {
+        Map<String, String> parameters = new TreeMap<>();
```

Comment:
Really small comment, I think this could just be `Map<String, String> parameters = Collections.singletonMap(INCLUDE_TYPE_NAME_PARAMETER, "true");`.